erpnext/buying/doctype/purchase_order/purchase_order.js

```diff
@@ -181,6 +182,12 @@ erpnext.buying.PurchaseOrderController = erpnext.buying.BuyingController.extend(
 				})
 			}, __("Add items from"));
 
+		cur_frm.add_custom_button(__('For Supplier'),
```

Comment:
`cur_frm` !!!!

Should be a button below supplier field only shown when document is not saved (__islocal)